notebooks/signac_101_Getting_Started.ipynb

```diff
@@ -29,7 +29,7 @@
   },
```

Comment:
<p>I know personal paths have already been discussed but in general do we want the output to be committed?</p><p>pros:</p><ul><li>with nbval this will act as a unit test</li><li>users can see the output without running on binder/static view is useful</li><li>if it's indexed by the docs, then when users search for errors/command output they will find an example</li><li>when we embed these notebooks they will look superb on rtd (like fresnel does)</li></ul><p>cons:</p><ul><li>it can make diffs hard</li><li>output could include personal information (like username)</li></ul><p> </p><p>I'm now thinking we include the output, but configure some cells to hide output when we commit, thoughts @bdice ? I know you raised this issue elsewhere</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I'm strongly in favor of committing the outputs for our tutorials -- seeing the outputs means a lot to new users who just want to understand what the framework does and how it should behave in practice. #25 fixes the issues with personal paths / metadata and makes it much easier to reproduce the notebooks' output (pending some issues that cause warnings in signac/signac-flow; see the description of #25 for details).

Diffs are hard, but much better with stripped metadata (fixed in #25). Hopefully I've mitigated the "cons" enough to warrant moving forward with committed output. (Now I sound like a motivational speaker for teams, since I'm throwing out phrases like "committed output." Synergy!)